### PR TITLE
Add variable for bucket suffixes for gitlab

### PR DESCRIPTION
**Pull request by corang**

### CODE REVIEW

**Review by justin-o12 — APPROVED** [review]
LGTM!  Once this is merged, we'll make sure our IaC lines up with the `uds-<bucket name><optioal prefix>` convention.

**Review by anthonywendt — APPROVED** [review]
LGTM

**Review by zachariahmiller — APPROVED** [review]
LGTM

**Pull request merged**
